app/web/features/search/SearchBox.tsx

```diff
@@ -139,7 +139,9 @@ export default function SearchBox({
           <RadioGroup
             className={classes.justifyContent}
             row
-            onChange={(event) => handleOnChangeRadioButton(event, searchType)}
+            onChange={(event, value) =>
+              handleOnChangeRadioButton(event, value as "location" | "keyword")
```

Comment:
Could you add a comment here that material-ui has this type as `unknown` hence the type coercion?